### ISSUE TITLE
Round trip changes the offset of DateTimeOffset

### ISSUE THREAD

**Issue by hvester** [fsprojects/FSharp.AWS.DynamoDB#6]
### Description

The offset information of DateTimeOffset is lost when it is put to DynamoDb.

### Repro steps

```
open System
open Amazon
open Amazon.DynamoDBv2
open FSharp.AWS.DynamoDB

let dynamoDbClient = new AmazonDynamoDBClient("AccessKey", "SecretKey", RegionEndpoint.EUWest1)

type TestTable =
    { [<HashKey>] Id : int
      Timestamp : DateTimeOffset }

let table = TableContext.Create<TestTable>(dynamoDbClient, tableName = "TestTable", createIfNotExists = true)

table.PutItem({ Id = 1; Timestamp = DateTimeOffset(2016, 1, 1, 0, 0, 0, TimeSpan(0, 0, 0))})
table.PutItem({ Id = 2; Timestamp = DateTimeOffset(2016, 1, 1, 0, 0, 0, TimeSpan(5, 0, 0))})

let r1 = table.Query(<@ fun r -> r.Id = 1 @>)
// val r1 : TestTable [] = [|{Id = 1;
//                            Timestamp = 1.1.2016 2:00:00 +02:00;}|]
let r2 = table.Query(<@ fun r -> r.Id = 2 @>)
// val r2 : TestTable [] = [|{Id = 2;
//                            Timestamp = 31.12.2015 21:00:00 +02:00;}|]

```
### Expected behavior

Offset should be the same as it was when the item was inserted.

### Actual behavior

Returned DateTimeOffset represents the same instant, but with the offset of local time zone.

### Known workarounds

Use string instead of DateTimeOffset.

### Related information 

* FSharp.AWS.DynamoDB version: 0.5.0-beta

**Comment by eiriktsarpalis** [comment]
Fixed in 0.6.0-beta

**Comment by hvester** [comment]
Great!
